### PR TITLE
DataGrid: Support Multiple items on DataGridSelectColumn

### PR DESCRIPTION
**Pull request by David-Moreira**

### CODE REVIEW

File: Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs [hunk 1]
```diff
@@ -18,6 +18,11 @@ public class BaseDataGridColumn<TItem> : BaseDataGridComponent
     /// <returns>Formatted display value.</returns>
     public string FormatDisplayValue( object value )
     {
+        if (value is string[] values )
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs:21]: **Reminder to self :** Can we perhaps try to just find whether it's a collection and enumerate over it while doing .ToString to support more types?

File: Source/Extensions/Blazorise.DataGrid/_DataGridCellSelectEdit.razor.cs [hunk 2]
```diff
@@ -73,6 +74,11 @@ private void OnSelectedValueChanged( object value )
         }
     }
 
+    private async Task OnSelectedValuesChanged( IReadOnlyList<object> values )
+    {
+        await CellValueChanged.InvokeAsync( values?.Select( x => x.ToString() )?.ToArray() );
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.DataGrid/_DataGridCellSelectEdit.razor.cs:79]: **Reminder to self :** Can we somehow support more types other then string[]? int[], so on...

**Comment by David-Moreira** [comment]
@stsrki Can you take a look at the current solution. I believe this should be fine. What do you think?

File: Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs [hunk 3]
```diff
@@ -18,6 +19,24 @@ public class BaseDataGridColumn<TItem> : BaseDataGridComponent
     /// <returns>Formatted display value.</returns>
     public string FormatDisplayValue( object value )
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs:20]: Should we make `FormatDisplayValue` virtual and override the logic in DataGridSelectColumn?

File: Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs [hunk 4]
```diff
@@ -18,6 +19,24 @@ public class BaseDataGridColumn<TItem> : BaseDataGridComponent
...
+            foreach ( var item in values )
+            {
+                if ( firstIteration )
+                {
+                    collectionDisplayValue = item?.ToString();
+                    firstIteration = false;
+                    continue;
+                }
+
+                collectionDisplayValue += $",{item}";
```
Comment (stsrki) [inline Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs:35]: I would use StringBuilder for concatenation.

**Pull request merged**